docfx_project/GitHooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+FILES_STAGED=`git diff --name-only --cached | wc -l`
+
+if [[ $FILES_STAGED -le 0 ]]
+then
+	exit
+fi
+
+echo "GENERATING DOCFX DOCUMENTATION"
+DOCFX="$HOME/.nuget/packages/docfx.console/2.56.6/tools/docfx.exe"
```

Comment:
this pre-commit hook assumes windows, but most devs at indico are running ubuntu. Is there another way to hook this in?